docs/migrating11to17.md

```diff
@@ -45,5 +45,7 @@ The following table lists the new OpenJ9 features and notable changes with the O
 
 | Features and changes  | OpenJ9 release|
 |-----------------------|---------------|
-|Linux builds for all platforms use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).| 0.33.0  |
-|The default operating system stack size on x64 platforms is increased from 256 KB to 512 KB to accommodate vector support. You can change the operating system stack size by using the [-Xmso](xmso.md) option.| 0.33.0   |
+|The default operating system stack size on x64 platforms is increased from 256 KB to 512 KB to accommodate vector support. You can change the operating system stack size by using the [-Xmso](xmso.md) option.|  0.33.0   |
+|When the VM loads an application native library (DLL), it searches for the library in the path that is specified in the `-Djava.library.path` system property or in the following environment variables:<ul><li>`LIBPATH` - z/OS&reg; and AIX&reg;</li><li>`LD_LIBRARY_PATH` - Linux&reg;<li>`PATH` - Windows&trade;</li></ul><br>In Java 11 and earlier versions, the VM searched for the application native libraries by using both the `-Djava.library.path` system property and the environment variable.<br>From Java 17 onwards, the search is based on only the `-Djava.library.path` system property value that is obtained when the VM starts. Changing the `-Djava.library.path` system property value after VM startup doesn't change the search. If the path is set on the command line by using the `-Djava.library.path` system property, only this path is used for the search. The path that is specified by the variable is not used to search for the application native libraries.<br>The default value of the `-Djava.library.path` system property is still derived from the environment variable.  |  0.29.1   |
```

Comment:
`-Djava.library.path` is a command line option. For all usage except for "If the path is set on the command line by using the `-Djava.library.path` system property", it should just be `java.library.path`. 
i.e. "it searches for the library in the path that is specified in the `java.library.path` system property"

Reply:
"by using the -Djava.library.path system property"
-> "by using the -Djava.library.path option"

Reply:
I assume "the variable" in "The path that is specified by the variable is not used to search for the application native libraries." refers to the LIBPATH, etc. variables? This isn't clear.

Reply:
If I say "... specified by the environment variables...", will that be clear?

Reply:
Sounds good.

Reply:
It should be "environment variable" rather than "environment variables" (no 's').